### ISSUE TITLE
Add Information about using hard links to relocate folders

### ISSUE THREAD

**Issue by furicle** [pmario/file-backups#5]
It seems like the biggest short coming of file-backups vs TiddlyFox is the forced location for files.

I *think* it's possible to use hard links to partially work around that.

It would be nice if we documented that proceedure not only in the README, but in the plug in itself.

I was hoping the plug in "knows" the location of the downloads folder, and the operating system it's running under.   That way we could have something like a button where the Dashboard button is now, that opens a dialog box giving instructions on the `mklink` or `ln` syntax that's correct for that users computer...

**Comment by pmario** [comment]
Yes, it's planned to add this info. Especially for windows users it's important, since "soft links" won't work. We need to use windows junctions.

**Comment by pmario** [comment]
Dock is improved in the next version, which will be up soon
